backend/substrapp/tasks/tasks.py

```diff
@@ -443,9 +446,15 @@ def prepare_tuple(subtuple, tuple_type):
         log_fail_tuple(tuple_type, subtuple['key'], error_code)
 
 
-@app.task(bind=True, ignore_result=False)
+@app.task(bind=True, ignore_result=False,
+          autoretry_for=(Exception,),
+          retry_kwargs={'max_retries': CELERY_TASK_MAX_RETRIES, 'countdown': CELERY_TASK_RETRY_DELAY_SECONDS})
 def compute_task(self, tuple_type, subtuple, compute_plan_id):
 
+    attempt = self.request.retries + 1
+    max_attempts = CELERY_TASK_MAX_RETRIES + 1
```

Comment:
why do we need to handle this manually? isn't celery autoretry with max_tries sufficient?

Reply:
Fixed: https://github.com/SubstraFoundation/substra-backend/pull/66/commits/0462f788d66e64ff1ccc02b9d751ac377b434ea2